src/Microsoft.Data.Analysis/GroupBy.cs

```diff
@@ -464,5 +466,12 @@ public override DataFrame Mean(params string[] columnNames)
             return ret;
         }
 
+        public IEnumerable<IGrouping<TKey, DataFrameRow>> Groupings
```

Comment:
Do we need a new `Groupings` property? Or could we just implement `IEnumerable<IGrouping<TKey, DataFrameRow>>` on `GroupBy<TKey>` itself?

Reply:
I think this should be implemented directly on the `GroupBy<TKey>` type too. Especially since this PR is now exposing a `GroupBy<TKey>` method

Reply:
@eerhardt, I agree with you, I also prefer not to have extra property. But there is a small problem here:  GroupBy class has it's own implementation of Count(), First() and other methods, that are provided by LINQ extensions to IEnumerable interface. Moreover all these methods have different return type. For example Count() returns dataframe and not Int. First()  also returns dataframe and not IGrouping. In this case, using of new LINQ method will require explıcıt casting:

`var grouping = df.GroupBy<string>("Parity");
DataFrame count  = grouping.Count(); // GroupBy class own method
int count  = ((IEnumerable<IGrouping<string, DataFrameRow>>)grouping).Count(); //LINQ method
`
Do you have any ideas how this can be solved?

Reply:
Ah, yea that is a problem. The `Count, First and other` methods are convenient APIs to inspect the `GroupBy` quickly. For now, I think we can create this new property. In the future, we can implement `IEnumerable<IGrouping<TKey, DataFrameRow>>` on `GroupBy<TKey>` and refactor the `Count` and `First` implementations to possibly just use equivalent  `Linq` methods instead. 

I wonder if we should name this property `Groups` instead of `Groupings`? @eerhardt

Reply:
> I wonder if we should name this property Groups instead of Groupings?

I don't think I have a strong opinion either way... Maybe since the interface is named `IGrouping`, calling this `Groupings` seems appropriate.